File: Modul 5/Bahasa Python/PRAK502-2310817120001-DinaIzzatiElfadheya.py
def mutlak(angka):
    if angka < 0:
        angka = -angka
    return angka

def hitung(nilai1, nilai2):
    hasil = nilai1 - nilai2
    return mutlak(hasil)

a, c, b, d= map(int, input("Input\n").split())
print("\nOutput")
hasil = hitung(a, b) + hitung(c, d)
print((hasil))

File: Modul 5/Bahasa C/PRAK502-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
#include<math.h>

int mutlak(int angka) {
    if(angka < 0) {
        angka=-angka;
    }
    return angka;
}
int hitung(int nilai1, int nilai2) {
    int Hasil= nilai1-nilai2;   
    return mutlak(Hasil);
}
int main() {
    int a, b, c, d;

    printf("Input\n");
    scanf("%d %d %d %d", &a, &c, &b, &d);

    int Hasil = hitung(a, b) + hitung(c, d);
    printf("\nOutput\n");
    printf("%d", mutlak(Hasil));

    return 0;
}

